grr/server/grr_response_server/gui/ui/components/flow_details/helpers/osquery_results_table.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+import { OsqueryTable } from '@app/lib/api/api_interfaces';
+
+/**
+ * Component that displays an OsqueryTable object as a HTML table.
+ */
+@Component({
+  selector: 'osquery-results-table',
+  templateUrl: './osquery_results_table.ng.html',
+  styleUrls: ['./osquery_results_table.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OsqueryResultsTable {
+  @Input()
+  table?: OsqueryTable;
```

Comment:
Please handle the nullability of `table`. This input shows table as potentially undefined, but the template expects it to be always set.